.changeset/stale-forks-tickle.md

```diff
@@ -0,0 +1,30 @@
+---
+"@fluidframework/tree": minor
+---
+---
+"section": tree
+---
+
+Added new and improved SharedTree branching API
+
+This refreshes the (currently alpha) SharedTree branching API to be more accessible and intuitive.
+The branching functionality (`branch`, `merge`, `rebaseOnto`, etc.) are now directly available on the view object rather than a separate object.
+In particular, `TreeViewAlpha` is now a `TreeContext`, which exposes the methods to coordinate branches.
+See the `TreeContext` interface for more details.
+
+```typescript
```

Comment:
Nit: it might be useful to give an example of how the same operation would have been achieved with the previous pattern for comparison. Gives readers a clearer path for migrating.

Reply:
Probably also worth noting the APIs that were deprecated in this PR.

Reply:
Agreed - I added some more detail in here.